### ISSUE TITLE
[Supporting the Platform-Make Code - Home Page]:  When 'Text Spacing' is applied, the paragraph heading is getting truncated.

### ISSUE THREAD

**Issue by monisverma** [microsoft/pxt-microbit#5462]
### **User impact:**
If any text is not appearing properly on the screen when text spacing is applied, the user with vision problems will be blocked as the user will not be able to see the text properly and will miss the information.

### **Note:**
User credentials should NOT be included in the bug.

### **Pre - Requisite:**

- Open https://chrome.google.com/webstore/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne?hl=en 
- Click on ‘Add to Chrome’ button.
- After adding, you should find Stylus icon on the tool bar, click it to open drop down menu.
- Select ‘Manage Styles’ from the menu.
- On the manage styles screen, click on ‘Create new Style’.
- The New style screen would open, here give a suitable name for your new style, e.g. ‘text spacing’.

**Add following CSS under ‘Code’ field –**
*{
     line-height: 1.5! important;
     letter-spacing: 0.12em! important;
     word-spacing: 0.16em! important;
}
p {
     margin-bottom: 2em !important;
}
Click on ‘Save’.

### **Repro Steps:**

1. Open the URL https://makecode.microbit.org/ 
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Turn on Stylus tool and verify whether the text is visible properly or not.

### **Actual Result:**
**Issue:** When 'Text Spacing' is applied, the paragraph heading is getting truncated.
**Observation:** When 'Text Spacing' is applied on the Home Page, the "Track the soil moisture of your plants!", "Introduction to the BBC micro:bit",  "Behind the MakeCode Hardware " text is getting truncated.

### **Expected Result:**
When 'Text Spacing' is applied, the all-page content should be visible properly.
Ex: When 'Text Spacing' is applied on the Home Page, the "Track the soil moisture of your plants!", "Introduction to the BBC micro:bit",  "Behind the MakeCode Hardware " text should visible properly.

### **ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:**
NA

### **UIA/DOM details available:** 
yes

### **MAS Reference:**
[MAS 1.4.12 – Text Spacing](https://aka.ms/MAS1.4.12)

### **Reference Links:** 

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier. 
- External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”. 
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### **Test Environment:**
**OS:** Windows 11 22H2 (OS Build:22621.3085).
**Browser:** Chrome Version 121.0.6167.86 (Official Build) (64-bit)
**URL:** https://makecode.microbit.org/
**Tools:** Developer tool, Stylus

### **Attachment:**
<img width="471" alt="image" src="https://github.com/microsoft/pxt-microbit/assets/158456802/f1b7addd-b291-48fd-930b-b2d29d7919f6">
<img width="487" alt="image" src="https://github.com/microsoft/pxt-microbit/assets/158456802/86d5e8aa-c117-43dd-8148-dbf77a08b75d">

**Comment by monisverma** [comment]
#Accessibility#A11yMAS;#A11ySev2;#E2E-HCL-MakeCode-Jan2024;#MAS1.4.12;#HCL;#Win11-Chrome;#benchmark
